scylla-cql/src/frame/value_tests.rs

```diff
@@ -41,6 +42,26 @@ fn serialized_only_new<T: SerializeCql>(val: T, typ: ColumnType) -> Vec<u8> {
     result
 }
 
+#[test]
+fn timeuuid_sorting() {
+    let sorted_timeuuid_vals: Vec<CqlTimeuuid> = vec![
+        CqlTimeuuid::from_str("00000000-0000-1000-8080-808080808080").unwrap(),
+        CqlTimeuuid::from_str("00000000-0000-1000-ffff-ffffffffffff").unwrap(),
+        CqlTimeuuid::from_str("00000000-0000-1000-0000-000000000000").unwrap(),
+        CqlTimeuuid::from_str("fed35080-0efb-11ee-a1ca-00006490e9a4").unwrap(),
+        CqlTimeuuid::from_str("00000257-0efc-11ee-9547-00006490e9a6").unwrap(),
+        CqlTimeuuid::from_str("ffffffff-ffff-1fff-ffff-ffffffffffef").unwrap(),
+        CqlTimeuuid::from_str("ffffffff-ffff-1fff-ffff-ffffffffffff").unwrap(),
+        CqlTimeuuid::from_str("ffffffff-ffff-1fff-0000-000000000000").unwrap(),
+        CqlTimeuuid::from_str("ffffffff-ffff-1fff-7f7f-7f7f7f7f7f7f").unwrap(),
+    ];
+
+    let mut driver_sorted_timeuuid_vals = sorted_timeuuid_vals.clone();
+    driver_sorted_timeuuid_vals.sort();
+
+    assert_eq!(sorted_timeuuid_vals, driver_sorted_timeuuid_vals);
+}
```

Comment:
I'd like to see an example where values are not already sorted

Reply:
Actually this test is reduntant, since the test contained in `scylla` crate already tests it.